plugins/windows-debug.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: windows-debug
+spec:
+  version: v0.1.1
+  homepage: https://github.com/jsturtevant/windows-debug
```

Comment:
it seems like the auth plugins are not imported https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#auth-plugins consider fixing

Reply:
My bad, I assumed this would be in Go, since it's intended for Windows, but I realize it targets Windows clusters/nodes.

Reply:
Yes I mostly use WSL so it is usable from windows in that respect.  I realize folks may want to run from powershell which is why I was considering re-writing but wanted some feedback.  I assume lots of folks just use WSL.